### ISSUE TITLE
Payment Module settings during development

### ISSUE THREAD

**Issue by voloships** [litecommerce/core#28]
Hi,
I'm developing Payment Module for Lite Commerce.
I don't know if it's bug or feature I found, but it's very annoying.
1. I created basic module structure and settings (workable for backend).
2. Uploaded to /litecommerce/ folder root regarding directories structure.
3. Re-built cache and enabled module.

Expected:
1. Appearance of module in payment modules list.
2. Appearance of module in xlite_payment_methods, xlite_payment_method_settings and xlite_payment_method_translations tables.

I spent some time in debug of my module before found the solution:
I need to:
1. I created basic module structure and settings (workable for backend).
2. Uploaded to /litecommerce/ folder root regarding directories structure.
3. Re-built cache and enabled module.
4. Enable developer mode in config.
5. Pack module.
6. Disable and remove module.
7. Install module from package got on step 5.

**Comment by svowl** [comment]
If you develop module on core.git repository deployed from github, you can use utility 'restoredb' to upload install.yaml files for all existing modules:

> cd xlite/src
> ./restoredb demo admin (if you use LiteCommerce with Drupal as frontend)
> ./restoredb standalone demo admin (if you use LiteCommerce standalone)

(restoredb should have executable permissions or you should run it as 'php restoredb...')

After that just open any LiteCommerce page to generate cache.

**Comment by voloships** [comment]
No, I'm using local fresh installed Lite Commerce installation, but thanks for your tip.

P.S.: Can I contact you somehow to clarify some moments regarding payment module development?
